webui/react/package.json

```diff
@@ -31,6 +31,7 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.0.1",
```

Comment:
we've been using `@ant-design/icons` without explicitly installing it. now its added

Reply:
@keita-determined It looks like we were using `4.7.0` and we are now defining version `5.0.1` any explicit reason for the version change?

Reply:
i dont think version really matters because npm nicely adjust version, but i changed it with v4.7.0